### ISSUE TITLE
[bug] Babel eslint shipped as a dependency

### ISSUE THREAD

**Issue by Timer** [postcss/postcss-selector-parser#156]
v5.0.0-rc.3 marked `babel-eslint` a dependency (`^8.2.3`).

**Comment by Timer** [comment]
Upgrades to this dependency are breaking Create React App's preflight check, so fixing this quick would be awesome. ❤️

**Comment by leipert** [comment]
Jup, definitely shipped: https://unpkg.com/postcss-selector-parser@5.0.0-rc.3/package.json

**Comment by leipert** [comment]
Also: https://github.com/postcss/postcss-selector-parser/compare/v5.0.0-rc.3...master#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L43

Could we get an rc.4 release?

**Comment by evilebottnawi** [comment]
@leipert already do request to developer how have `npm` access

/cc friendly ping @jonathantneal

**Comment by leipert** [comment]
Thank you very much 🙇

**Comment by OKNoah** [comment]
How are people getting around this now?

**Comment by leipert** [comment]
@OKNoah If you use yarn, you can make use of [resolutions](https://yarnpkg.com/lang/en/docs/selective-version-resolutions/)

If you have `babel-loader`, you could dedupe it:
```json
{
  "dependencies": {
    "postcss-selector-parser": "^5.0.0-rc.3",
    "babel-eslint": "^10.0.1"
  },
  "resolutions": {
    "**/postcss-selector-parser/babel-eslint": "^10.0.1"
  }
}
```
or, if you don't use it, you can put in some random package url:
```json
{
  "dependencies": {
    "postcss-selector-parser": "^5.0.0-rc.3"
  },
  "resolutions": {
    "**/postcss-selector-parser/babel-eslint": "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz"
  }
}
```

**Comment by Timer** [comment]
Any updates for this?

**Comment by Pajn** [comment]
This broke my environment too now. Please, the total time for every developer hitting and debugging this error is growing. Checking out the old commit, running yarn remove and yarn publish takes five minutes. You do not have to be ready with the next version, just check out the previously published commit.

**Comment by Timer** [comment]
Yeah, we're starting to see issues pop up for this.
I respect that the developers of this project are likely busy, but if time can be made for this, that'd be great!

In the meantime, maybe we need to use a fork or downgrade (if possible) in Create React App.

**Comment by evilebottnawi** [comment]
You can use `master`, this already solved

**Comment by jonathantneal** [comment]
@evilebottnawi, can we publish even with the test issues?

**Comment by Pajn** [comment]
The problem is not that it can't be worked around. It's that is cost hours to find out why the babel build is all broken without any usable error messages all of a sudden.

**Comment by leipert** [comment]
@jonathantneal Given that the solution in https://github.com/postcss/postcss-selector-parser/pull/163 is to remove the failing tests, I would say it is safe to publish 👍

**Comment by evilebottnawi** [comment]
@jonathantneal yes and we need, it is problem on `stylelint` side, better drop `stylelint` testы https://github.com/postcss/postcss-selector-parser/pull/163 they outdated and give us nothing

**Comment by jonathantneal** [comment]
Okay, diving in.

**Comment by jonathantneal** [comment]
After many failed attempts at updating the babel-related devDependencies, I’m giving up and moving on to leaving the others updated and publishing.

**Comment by jonathantneal** [comment]
This is resolved in https://github.com/postcss/postcss-selector-parser/releases/tag/5.0.0-rc.4

I’ll provide one more update and close this issue once I’m able to publish the new release to npm, which should be very soon. Thanks @evilebottnawi, @Timer, @ai.

**Comment by jonathantneal** [comment]
Published as 5.0.0-rc.4.
